include/nighthawk/adaptive_load/metrics_plugin.h

```diff
@@ -11,6 +11,14 @@
 
 namespace Nighthawk {
 
+// Describes the period in which a Metrics plugin should measure.
+struct MeasuringPeriod {
+  // The beginning of the time period where nighthawk is measuring.
```

Comment:
We should expand these comments as well. It isn't clear what "where nighthawk is measuring" means, since the measuring is done by the plugin.

We should explain that this is the start time of the latest (current) iteration of Nighthawk test in the adaptive stage (maybe link to [the docs](https://github.com/envoyproxy/nighthawk/blob/main/docs/root/adaptive_load_controller.md) for the adaptive load controller).

Reply:
Done.